pointgrey_camera_driver/debian/postinst

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+#DEBHELPER#
+
+# A udev rules file was added during install, load and trigger rules
+if [ "$1" = configure ]; then
+  if which udevadm > /dev/null; then
+    set +e # Disable exit on error
+    udevadm control --reload-rules
+    service udev restart
+    udevadm trigger
+    set -e # Re-enable exit on error
+  fi
+fi
```

Comment:
I have ```postinst``` and ```postrm``` scripts that modify grub to set ```usbcore.usbfs_memory_mb=1000``` at boot. This is recommended in the [troubleshooting](http://wiki.ros.org/pointgrey_camera_driver#Troubleshooting) section of the wiki. Should I provide a PR?